src/unitxt/operator.py

```diff
@@ -135,8 +135,12 @@ class MultiStreamOperator(StreamingOperator):
 
     caching: bool = NonPositionalField(default=None)
 
-    def __call__(self, multi_stream: Optional[MultiStream] = None) -> MultiStream:
+    def __call__(
+        self, multi_stream: Optional[MultiStream] = None, **kwargs
```

Comment:
How is this change related? What does it do? Not clear.

Reply:
Completely unrelated will delete that

Reply:
done